bundle/manifests/o2ims.oran.openshift.io_inventories.yaml

```diff
@@ -67,6 +67,19 @@ spec:
                 required:
                 - enabled
                 type: object
+              artifactsServerConfig:
+                default:
```

Comment:
Outside of perhaps lab debugging is there a scenario where we need this switch  vs just unconditionally starting the server ?

Reply:
I followed the model of the other servers that have this `enabled` switch which is by default set to `true`. I did add the server in the `default` Inventory CR, so when the O-Cloud manager is installed, the artifacts server will automatically start.

Reply:
Ya, I agree... which should work towards removing these "enabled" boolean flags.  No idea outside of debugging.

Reply:
i would leave it as-is for now and remove them all in one shot later.